pyne/r2s.py

```diff
@@ -231,30 +231,42 @@ def total_photon_source_intensity(m, tag_name, sub_voxel=False):
             intensity += vol * np.sum(sv_data)
     return intensity
  
-def tag_e_bounds(m, e_bounds, tag_name='e_bounds'):
-    """This function tags the energy boundaries to the PyMOAB mesh
+def tag_phtn_src_info(m, value, tag_name, dtype=float):
+    """This function tags the information of photon source to the PyMOAB mesh
     instance as a sparse tag for the purpose of photon source sampling.
 
     Parameters
     ----------
     m : PyNE Mesh
        The mesh-based photon emission density distribution in p/cm3/s.
-    e_bounds : numpy array or list
-        The energy boundaries of a photon source.
+    value : numpy array, list or float
+        The information to be tagged, could be:
+        - e_bounds: The energy boundaries of a photon source, numpy array or
+            list, unit: eV.
+        - decay_time: The decay time after shutdown, unit: s.
+        - total_photon_source_intensity: The total photon source intensity at
+            current decay time, unit: p/s.
+        - version: The PyNE version that changes the information or
+            organization of the 'source.h5m'. Eg. use [0, 7, 3] for PyNE
+            release 0.7.3.
```

Comment:
No need to list the version value here. When the version changes, we'll likely forget to update this and it may confuse future developers reading through the code.